src/composable/Questionnaire.ts

```diff
@@ -172,19 +173,22 @@ export class Questionnaire implements IQuestionnaire {
    * @returns
    */
   getProgressPercent(props: IStepData, config: IQuestionableConfig): number {
+    if (matches(props.step?.sectionId, PAGE_TYPE.RESULTS)) {
+      return 100;
```

Comment:
100 or 1.0?

Reply:
If we're at the results section, we're 100% complete--no further calc required.